api/src/main/java/org/openmrs/OrderSetMember.java

```diff
@@ -0,0 +1,132 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+import java.io.Serializable;
+
+/**
+ * @since 1.12
+ * OrderSetMember
+ */
+public class OrderSetMember extends BaseOpenmrsMetadata implements Serializable {
```

Comment:
I notice that sortWeight has been removed from `OrderSetMember`. So, we're leaving sequencing to hibernate? That's fine. Just want to be sure.